data/css/modules/card/_media.scss

```diff
@@ -1,35 +1,30 @@
 .CardMedia {
-    padding: 10px;
     background-color: white;
 
-    separator {
-        background-color: $black-20-color;
-        min-height: 1px;
-    }
-
     &__caption {
-        font-size: .81em;  /* 13pt * 1/12 * 0.75 DPI correction factor */
-        font-family: $display-font;
+        font-size: 20px;
```

Comment:
Drive by review with no actual knowledge - but do we want to keep using em's rather than px here?